plugins/inputs/t128_graphql/t128_graphql.go

```diff
@@ -25,7 +24,10 @@ const (
 	DefaultRequestTimeout = time.Second * 5
 
 	//DefaultDeadline is the time for the graphQL server to build the response. Default is no deadline.
-	DefaultDeadline = time.Second * math.MaxInt32
+	DefaultDeadline = time.Second * 0
+
+	//MinTimeoutDeadlineDiff is the minimum value of plugin.Timeout - plugin.Deadline
+	MinTimeoutDeadlineDiff = time.Second * 3
```

Comment:
3 seconds seems excessive to me. I think leaving it at a second would be enough just to make sure the user is accounting for the possibility.